website/docs/components/copy/snippet/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+{description}
```

Comment:
@andgen404 had these not been filled in prior to my changes? (want to make sure I didn't miss something)

Reply:
No I didn't have these filled in, I deleted them on my branch.